.changeset/lemon-singers-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cmpsr/components": minor
+---
+
+implemente close button component
```

Comment:
at the beginning I had problems since we chakra closeButton component uses a different icon, after setting the exactly same styles from the figma specs it looked a little bigger, so I was chatting with Rik and he said that maybe the Icon was the problem, and that by chance @guilean was working with the icons and that i could talk to him, so he said me that we need to implement it from scrach via IconButton, that was the reason why it was implemented that way. 
After reviewing some PRs, I saw that another alternative could be using the Chakra CloseButton but this time passing the new Icon as children. Let me know if you want to make this change or you prefer to leave it like that.